cypress/integration/work-order/attend/close.spec.js

```diff
@@ -77,11 +77,235 @@ describe('Closing my own work order', () => {
     cy.loginWithOperativeRole()
   })
 
-  context.only('during normal working hours', () => {
+  context('during normal working hours', () => {
     beforeEach(() => {
       cy.clock(new Date(now).setHours(12, 0, 0))
     })
 
+    it('shows a validation error when no reason is selected', () => {
```

Comment:
For E2E tests, I'd suggest combining the validation error test cases into a single case for showing appropriate validation errors. These seem too focused for e2e tests